package-lock.json

```diff
@@ -1,9234 +1,15 @@
 {
   "name": "findologic",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
As discussed on slack, please update your npm version. It also states here that you downgraded.